pipelines/batch/src/main/java/org/openmrs/analytics/HapiRowDescriptor.java

```diff
@@ -43,4 +48,6 @@ static HapiRowDescriptor create(
   abstract String resourceVersion();
 
   abstract String jsonResource();
+
+  List<ResourceTag> tags;
```

Comment:
Add a quick one liner:

// Could be FHIR tags, or FHIR Profile information.

Reply:
sure